brightlayer-ui/doc-it
# Use externally linked PDF for icon guideline

Fixes https://github.com/brightlayer-ui/doc-it/pull/577#issuecomment-1211130472


#### Changes proposed in this Pull Request:
- Use github raw content instead of imported PDF


#### To Test:
- yarn start
- go to http://localhost:3000/style/icon-guidelines#follow-the-style-guide
- Download PDF using "Printable icon sketch template" card
- Should download
- When this get merged to dev, the 404 should not occur again
